src/custom/hooks/useIsOnline.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback, useEffect, useState } from 'react'
+
+export function isOnline() {
+  return window.navigator.onLine
```

Comment:
if it doesn't exist you should deal with it

Reply:
I was going to do it, but look at the types and browser acceptance: https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/onLine

Reply:
https://github.com/gnosis/cowswap/issues/818

Reply:
if this doesn't exist in someone's client this will return `undefined` which in the `if` check will show offline @anxolin